source/renderer/app/i18n/locales/en-US.json

```diff
@@ -132,7 +132,7 @@
   "global.language.german": "German",
   "global.language.japanese": "Japanese / 日本語",
   "global.language.korean": "Korean",
-  "global.passwordInstructions": "The password needs to be at least <strong>10 characters</strong> long, and have at least <strong>1 uppercase letter</strong>, <strong>1 lowercase letter</strong> and <strong>1 number</strong>.",
+  "global.passwordInstructions": "The password needs to be at least <strong>7 characters</strong> long, and have at least <strong>1 uppercase letter</strong>, <strong>1 lowercase letter</strong> and <strong>1 number</strong>.",
```

Comment:
@DeeJayElly please change "7" to "10" characters long.
@a-rukin this was wrongly presented in the designs - please update the designs!

Reply:
Fixed